### PR TITLE
KWoC Templates

### PR DESCRIPTION
**Pull request by lmanoj42**
### Refer the following points to add them according to the convention set for templates:

- Follow the guidelines in [about templates](https://github.com/kossiitkgp/mailing-scripts/blob/main/templates/README.md).
- See [selections templates](https://github.com/kossiitkgp/mailing-scripts/tree/main/templates/selections) for already existing examples.
- Update [varaible_mappings.py](https://github.com/kossiitkgp/mailing-scripts/blob/main/templates/variable_mappings.py)

## TODO
We send the following mails during KWoC, hence templates in [templates](https://github.com/kossiitkgp/mailing-scripts/tree/main/templates/) are required to created.

  - [ ] KWoC intro seminar publicity
  - [ ] Invite Projects
  - [ ] To previous participants, that KWoC is back
  - [x] Project Status mail
     - [x] Rejected and why
     - [x] Accepted
  - [ ] Coding Period has started 
  - [x] Mid evals deadline reminder
  - [x] Mid evals
    - [x] Qualified
    - [x] Disqualified
  - [x] End evals deadline reminder
  - [x] End evals
    - [x] Disqualified
    - [x] Mail with certificates' link to qualified contributors
    - Top Contributor with their swag details, if any, and certificate link - try to add a template, but this mail is highly probable to change every year..

### CODE REVIEW

File: templates/kwoc/mid-eval-reminder.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<b>KWoC 2023<b> : Mid-Evaluation Reminder!!
+
+Dear Participant,<br><br>
+
+Exciting times are ahead as we approach the mid-evaluation phase of Kharagpur Winter of Code (KWoC) 2023! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
+
+This is to remind you about the upcoming mid-evaluation for the project. Please ensure to submit your report by <b>{mid_eval_date}<b>, covering completed tasks and current status.<br><br>
```
Comment (proffapt) [inline templates/kwoc/mid-eval-reminder.txt]: - mid-evaluation is not specific to a project, it's of the student and student can be working on multiple projects
- ther's not such thing like report, they just have to do one commit/PR

File: templates/kwoc/end-eval-reminder.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+KWoC 2023 : Final Evaluation Reminder!!
```
Comment (proffapt) [inline templates/kwoc/end-eval-reminder.txt]:
```suggestion
KWoC 2023 : End-Evaluation Reminder!!
```

File: templates/kwoc/end-eval-reminder.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+Dear Participant,<br><br>
+
+We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project<br><br>
+
+This is to remind you that the final evaluation deadline for the project in the <b>Kharagpur Winter of Code (KWoC)<b> program is fast approaching.<br><br>
```
Comment (proffapt) [inline templates/kwoc/end-eval-reminder.txt]:
```suggestion
This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code (KWoC)<b> program is fast approaching.<br><br>
```

File: templates/kwoc/end-eval-reminder.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Please submit your report by {end_eval_date}, highlighting your progress, achievements, challenges, and suggestions.<br><br>
```
Comment (proffapt) [inline templates/kwoc/end-eval-reminder.txt]: There is no such thing like report

File: templates/kwoc/KWoC is back.txt [hunk 5]
```diff
@@ -0,0 +1,20 @@
+KWoC is back!! : It's time for another exciting coding journey
+
+Hello,<br><br>
+
+We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and We would like to invite you to be part of this amazing coding extravaganza once again!!<br><br>
```
Comment (proffapt) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and we would like to invite you to be part of this amazing coding extravaganza once again!!<br><br>
```

File: templates/kwoc/KWoC is back.txt [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+Your active involvement in previous editions of KWOC has contributed significantly to its success, and we genuinely appreciate your dedication.<br><br>
+
+<b>Mark your calenders for KWoC 2023 :<b><br><br>
```
Comment (proffapt) [inline templates/kwoc/KWoC is back.txt]: - Make 2023 the year as a variable, so that we can use it every year

**Review by proffapt — CHANGES_REQUESTED** [review]
- evaluations are not about `project`
- don't hard code the year

File: templates/kwoc/mid-eval-reminder.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<b>KWoC <b> : Mid-Evaluation Reminder!!
+
+Dear Participant,<br><br>
+
+Exciting times are ahead as we approach the mid-evaluation phase of Kharagpur Winter of Code (KWoC) 2023! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
```
Comment (proffapt) [inline templates/kwoc/mid-eval-reminder.txt]:
```suggestion
Exciting times are ahead as we approach the mid-evaluation phase of Kharagpur Winter of Code! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
```

File: templates/kwoc/mid-eval-reminder.txt [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+This is to remind you about the upcoming mid-evaluation for the project.<br><br>
```
Comment (proffapt) [inline templates/kwoc/mid-eval-reminder.txt]:
```suggestion
This is to remind you about the upcoming mid-evaluation.<br><br>
```

**Review by proffapt — CHANGES_REQUESTED** [review]
Date bhi toh bataoge mid-evals aur end-evals ki.

**Comment by harshkhandeparkar** [comment]
@manojxplores please mark the conversations as resolved if you have solved the issues.

**Comment by proffapt** [comment]
> @manojxplores please mark the conversations as resolved if you have solved the issues.

take updates from him.

File: templates/kwoc/KWoC is back.txt [hunk 9]
```diff
@@ -0,0 +1,20 @@
+KWoC is back!! : It's time for another exciting coding journey
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
KWoC is back! It's time for another exciting coding journey.
```

File: templates/kwoc/KWoC is back.txt (hunk 5 above)
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and we would like to invite you to be part of this amazing coding extravaganza once again!<br><br>
```

File: templates/kwoc/KWoC is back.txt [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+Hello,<br><br>
+
+We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and We would like to invite you to be part of this amazing coding extravaganza once again!!<br><br>
+
+Your active involvement in previous editions of KWOC has contributed significantly to its success, and we genuinely appreciate your dedication.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
Your active involvement in previous editions of KWoC has contributed significantly to its success, and we genuinely appreciate your dedication.<br><br>
```

File: templates/kwoc/KWoC is back.txt [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+<b>Mark your calenders for KWoC {year} :<b><br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
<b>Mark your calenders for KWoC {year}:<b><br><br>
```

File: templates/kwoc/KWoC is back.txt [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+<i>Registrations for students as well as projects</i> : {date}
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
<i>Registrations for students as well as projects</i>: {date}
```

File: templates/kwoc/KWoC is back.txt [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+<i>coding peroid</i>  : {start_date} to {end_date}<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
<i>Coding period</i>: {start_date} to {end_date}<br><br>
```

File: templates/kwoc/KWoC is back.txt [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+
+Stay tuned to our website and social media channels for updates on project selection, mentors, and other important announcements.<br><br>
+
+We eagerly anticipate your participation in KWOC {year}. Let's make this edition even more remarkable!
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]:
```suggestion
We eagerly anticipate your participation in KWoC {year}. Let's make this edition even more remarkable!
```

File: templates/kwoc/KWoC is back.txt [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+We hope this email finds you well. We are thrilled to announce that <b>Kharagpur Winter of Code<b> is back this year, and We would like to invite you to be part of this amazing coding extravaganza once again!!<br><br>
+
+Your active involvement in previous editions of KWOC has contributed significantly to its success, and we genuinely appreciate your dedication.<br><br>
+
+<b>Mark your calenders for KWoC {year} :<b><br><br>
+
+<i>Registrations for students as well as projects</i> : {date}
+<i>coding peroid</i>  : {start_date} to {end_date}<br><br>
+
+Stay tuned to our website and social media channels for updates on project selection, mentors, and other important announcements.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]: I think we can add links to the KWoC Slack and Discord here. And maybe even a link to our fb, insta, twitter, and linkedin.

File: templates/kwoc/KWoC is back.txt [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+
+We eagerly anticipate your participation in KWOC {year}. Let's make this edition even more remarkable!
+
+ 
+
+
```
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt]: These empty lines seem unnecessary.

File: templates/kwoc/end-eval-reminder.txt [hunk 17]
```diff
@@ -0,0 +1,12 @@
+KWoC : End-Evaluation Reminder!!
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-reminder.txt]:
```suggestion
<b>KWoC End-Evaluation Reminder</b>
```

File: templates/kwoc/end-eval-reminder.txt [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+
+Dear Participant,<br><br>
+
+We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-reminder.txt]:
```suggestion
We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project.<br><br>
```

File: templates/kwoc/end-eval-reminder.txt [hunk 19]
```diff
@@ -0,0 +1,12 @@
...
+
+This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code<b> program is fast approaching. The deadline for the end-evaluation is {end_eval_date}.<br><br>
+
+Once again, thank you for your active involvement in KWoC. Your contributions have been valuable in making this program a success.
+
+
+
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-reminder.txt:10]: Same here. Is there any reason for adding these empty lines?

File: templates/kwoc/end-eval-reminder.txt [hunk 20]
```diff
@@ -0,0 +1,12 @@
+KWoC : End-Evaluation Reminder!!
+
+Dear Participant,<br><br>
+
+We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project<br><br>
+
+This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code<b> program is fast approaching. The deadline for the end-evaluation is {end_eval_date}.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-reminder.txt]: I think we should mention that the participant should check their dashboard to complete the end evaluation.

File: templates/kwoc/mid-eval-qualified.txt [hunk 21]
```diff
@@ -0,0 +1,14 @@
+KWoC : Congratulations on Mid-Evaluation Success!
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-qualified.txt]:
```suggestion
KWoC: Congratulations on Mid-Evaluation Success!
```

File: templates/kwoc/mid-eval-qualified.txt [hunk 22]
```diff
@@ -0,0 +1,14 @@
...
+
+
+Dear participant,<br><br>
+
+Congratulations on your successful mid-evaluation for the project as part of the <b>Kharagpur Winter of Code (KWoC)<b> program!<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-qualified.txt]:
```suggestion
Congratulations on your completing your mid-evaluation in <b>Kharagpur Winter of Code (KWoC) {year}<b> program!<br><br> 
```

File: templates/kwoc/mid-eval-qualified.txt [hunk 23]
```diff
@@ -0,0 +1,14 @@
+KWoC : Congratulations on Mid-Evaluation Success!
+
+
+Dear participant,<br><br>
+
+Congratulations on your successful mid-evaluation for the project as part of the <b>Kharagpur Winter of Code (KWoC)<b> program!<br><br> 
+
+Your hard work and dedication have truly paid off, and we are thrilled with your achievements.<br><br>
+
+Thank you for your valuable contributions to the project's progress. Your efforts have been outstanding, and we look forward to your continued excellence in the upcoming phases of the KWoC program.
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-qualified.txt]:
```suggestion
Thank you for your valuable contributions to the project's progress. Your efforts have been outstanding, and we look forward to your continued excellence in the upcoming phases of the program.
```

File: templates/kwoc/mid-eval-reminder.txt [hunk 24]
```diff
@@ -0,0 +1,11 @@
+<b>KWoC <b> : Mid-Evaluation Reminder!!
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-reminder.txt]:
```suggestion
<b>KWoC: Mid-Evaluation Reminder</b>
```

File: templates/kwoc/mid-eval-reminder.txt [hunk 25]
```diff
@@ -0,0 +1,11 @@
...
+
+Dear Participant,<br><br>
+
+Exciting times are ahead as we approach the mid-evaluation phase of Kharagpur Winter of Code! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
+
+This is to remind you about the upcoming mid-evaluation, which is scheduled to take place on {mid_eval_date}.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-reminder.txt]: Again, mention the dashboard.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: templates/kwoc/KWoC is back.txt
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC is back.txt:1]: Please remove the spaces in the name of this file. `kwoc-is-back.txt` is better since it will be used in a terminal.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: templates/kwoc/KWoC-is-back.txt
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC-is-back.txt:1]: Keep the filename in all small letters if possible

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: templates/kwoc/KWoC-is-back.txt
Comment (harshkhandeparkar) [inline templates/kwoc/KWoC-is-back.txt:1]: Please use only lowercase letters in the filename for consistency.

File: templates/kwoc/end-eval-disqualified.txt [hunk 26]
```diff
@@ -0,0 +1,9 @@
+KWoC: End-Evaluation Result
+
+Dear participant,<br><br>
+
+We regret to inform you that your contributions in KWoC were not up to the mark.We appreciate your active participation in the Kharagpur Winter of Code (KWoC) program. Your efforts have not gone unnoticed, and we value your dedication to open source development.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-disqualified.txt]:
```suggestion
We regret to inform you that your contributions in KWoC were not up to the mark. We appreciate your active participation in the Kharagpur Winter of Code (KWoC) program. Your efforts have not gone unnoticed, and we value your dedication to open source development.<br><br>
```

File: templates/kwoc/end-eval-qualified.txt [hunk 27]
```diff
@@ -0,0 +1,14 @@
+KWoC: End-Evaluation Result
+
+Dear participant,<br><br>
+
+We hope this message finds you in good spirits.We're delighted to inform you that, you have successfully passed the end evaluations of the Kharagpur Winter of Code {year}.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-qualified.txt]:
```suggestion
We hope this message finds you in good spirits. We're delighted to inform you that you have successfully passed the end evaluations of the Kharagpur Winter of Code {year}.<br><br>
```

File: templates/kwoc/end-eval-qualified.txt [hunk 28]
```diff
@@ -0,0 +1,14 @@
...
+
+Your patience and dedication during this evaluation phase have not gone unnoticed. The evaluation process was challenging, and we admire your hard work and dedication in contributing to KWoC.<br><br>
+
+Once again, congratulations on your successful completion of the Kharagpur Winter of Code.we extend our best wishes for your upcoming pursuits<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-qualified.txt]:
```suggestion
Once again, congratulations on your successful completion of the Kharagpur Winter of Code. We extend our best wishes for your upcoming pursuits.<br><br>
```

File: templates/kwoc/end-eval-qualified.txt [hunk 29]
```diff
@@ -0,0 +1,14 @@
...
+
+Please find your certificate here: {certificate-link}<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/end-eval-qualified.txt]:
```suggestion
Please find your certificate here: {certificate_link}<br><br>
```
Best not to use `-`s in variables :)

File: templates/kwoc/mid-eval-reminder.txt [hunk 30]
```diff
@@ -0,0 +1,10 @@
+<b>KWoC: Mid-Evaluation Reminder!!</b>
+
+Dear Participant,<br><br>
+
+Exciting times are ahead as we approach the mid-evaluation phase of Kharagpur Winter of Code! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
+
+This is to remind you about the upcoming mid-evaluation, which is scheduled to take place on {mid_eval_date}.<br><br>
```
Comment (harshkhandeparkar) [inline templates/kwoc/mid-eval-reminder.txt]: It would be better to rephrase this as a deadline.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by harshkhandeparkar — DISMISSED** [review]

File: templates/kwoc/mid-eval-reminder.txt [hunk 31]
```diff
@@ -0,0 +1,10 @@
+<b>KWoC: Mid-Evaluation Reminder</b>
+
+Dear Participant,<br><br>
+
+Exciting times are ahead as we approach the mid-evaluation of Kharagpur Winter of Code! Your journey so far has been remarkable, and we can't wait to see the progress you've made.<br><br>
+
+This is to remind you that the deadline to complete your mid-evaluation requirements is on {mid_eval_date}.<br><br>
+
+Don't forget to check your dashboard to view your mid-evaluation status.<br><br>
+<em>Keep up the excellent work, and let's make this mid-evaluation a success to remember!<em>
```
Comment (proffapt) [inline templates/kwoc/mid-eval-reminder.txt:10]: Add this into end eval reminder..

**Review by proffapt — CHANGES_REQUESTED** [review]
- For disqualified templates it would be great if we can add some motivation links.. like we have during our selection process.. can be those exact same things and maybe some other if you find.
- kwoc-is-back has to be a good full fledged page.. something like this..
   image

File: templates/kwoc/mid-eval-reminder.txt (hunk 31 above)
Comment (harshkhandeparkar) [reply templates/kwoc/mid-eval-reminder.txt:10]: Usme hai sir

Comment (proffapt) [reply templates/kwoc/mid-eval-reminder.txt:10]: aaahh, ok then

**Comment by harshkhandeparkar** [comment]
Should we convert all templates to html files? @proffapt ?

File: templates/kwoc/kwoc-is-back.html [hunk 32]
```diff
@@ -0,0 +1,129 @@
...
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,300&display=swap');
+        </style>
+        <div
+            style="background-color: hsl(225, 19%, 8%); color: white; max-width: 600px; margin-inline: auto; margin: 1rem auto;">
+            <div
+                style="padding: 1rem; background-image: url('https://i.ibb.co/zRrxwMT/koss-logo-bg-dark-rounded-1-2-removebg-preview.png'), url('https://i.ibb.co/zfSVzq7/foss.png'), linear-gradient(#000, #0009, #0001), url('https://i.ibb.co/mC0XG8H/bg.jpg'); background-position: 1rem 1rem, calc(100% - 1rem) 1rem, center, center; background-repeat: no-repeat, no-repeat, no-repeat, no-repeat; background-size: 3rem, 3rem, cover, cover; text-align: center; height: 300px;">
+                <div style="display: inline-block; vertical-align: middle; padding: 7rem 0;">
+                    <h1 style="text-align: center; margin: 0; font-size: 2rem; font-family: 'Fira Code', monospace; font-family: 'Roboto', sans-serif;">Kharagpur Winter Of Code</h1>
+                    <h2 style="text-align: center; font-family: 'Roboto', sans-serif;"><b>Is Back !!</b></h2>
```
Comment (harshkhandeparkar) [inline templates/kwoc/kwoc-is-back.html]:
```suggestion
                    <h2 style="text-align: center; font-family: 'Roboto', sans-serif;"><b>Is Back!</b></h2>
```

File: templates/kwoc/kwoc-is-back.html [hunk 33]
```diff
@@ -0,0 +1,129 @@
...
+                </div>
+            </div>
+
+            <div style="padding: 2rem; color: #81c6e4; background-color: #010b1a;">
+                <p style="font-size: 2rem; margin: 0; margin-bottom: 0.75rem; text-align: center;"><b>Hello!</b></p>
+                <p style="margin: 0; margin-bottom: 1rem;">We hope this email finds you well. We are thrilled to
+                    announce
+                    that Kharagpur Winter
```
Comment (harshkhandeparkar) [inline templates/kwoc/kwoc-is-back.html]: Why are there newlines here?

File: templates/kwoc/kwoc-is-back.html [hunk 34]
```diff
@@ -0,0 +1,129 @@
...
+                    back this year, and we would like to invite you to be part of this amazing coding extravaganza
+                    once again</p>
+                <p style="margin: 0; margin-bottom: 1rem;">Your active involvement in previous editions of KWoC has
+                    contributed significantly to
+                    its success, and we
+                    genuinely appreciate your dedication.</p>
+
+                <p style="margin: 0; ">Mark your calenders for KWoC {year}</p>
+                <p style="margin: 0;">Registrations for students as well as projects: {date}</p>
+                <p style="margin: 0; margin-bottom: 1rem;">coding peroid: {start_date} to {end_date}</p>
```
Comment (harshkhandeparkar) [inline templates/kwoc/kwoc-is-back.html]:
```suggestion
                <p style="margin: 0; margin-bottom: 1rem;"><b>Coding period</b>: {start_date} to {end_date}</p>
```

File: templates/kwoc/kwoc-is-back.html [hunk 35]
```diff
@@ -0,0 +1,129 @@
...
+                    of Code is
+                    back this year, and we would like to invite you to be part of this amazing coding extravaganza
+                    once again</p>
+                <p style="margin: 0; margin-bottom: 1rem;">Your active involvement in previous editions of KWoC has
+                    contributed significantly to
+                    its success, and we
+                    genuinely appreciate your dedication.</p>
+
+                <p style="margin: 0; ">Mark your calenders for KWoC {year}</p>
+                <p style="margin: 0;">Registrations for students as well as projects: {date}</p>
```
Comment (harshkhandeparkar) [inline templates/kwoc/kwoc-is-back.html]:
```suggestion
                <p style="margin: 0;"><b>Registrations for students as well as projects</b>: {date}</p>
```

File: templates/kwoc/kwoc-is-back.html [hunk 36]
```diff
@@ -0,0 +1,129 @@
...
+            </div>
+
+            <div style="padding: 2rem; color: #81c6e4; background-color: #010b1a;">
+                <p style="font-size: 2rem; margin: 0; margin-bottom: 0.75rem; text-align: center;"><b>Hello!</b></p>
+                <p style="margin: 0; margin-bottom: 1rem;">We hope this email finds you well. We are thrilled to
+                    announce
+                    that Kharagpur Winter
+                    of Code is
+                    back this year, and we would like to invite you to be part of this amazing coding extravaganza
+                    once again</p>
```
Comment (harshkhandeparkar) [inline templates/kwoc/kwoc-is-back.html]:
```suggestion
                    once again.</p>
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by proffapt** [comment]
> Should we convert all templates to html files? @proffapt ?

They are already HTML right?
If you mean like such bg and stuff.. if there is time then isse behtar kya hoga..

**Comment by harshkhandeparkar** [comment]
> > Should we convert all templates to html files? @proffapt ?
> 
> They are already HTML right? If you mean like such bg and stuff.. if there is time then isse behtar kya hoga..

Mera matlab tha `.html` daal dete hain sab mein.

**Comment by harshkhandeparkar** [comment]
Also haan, baaki KWoC related templates mein bg daal dete hain.

**Comment by proffapt** [comment]
> Mera matlab tha .html daal dete hain sab mein.

Pehle itna hi karlo, kuch farak nahi padta anyhow. It should just work in the starting.

**Review by harshkhandeparkar — DISMISSED** [review]
check slack @manojxplores
